updated importing-pdfs page (#47)
iXtenda/iVP-gitbook: home/ivp-planning/getting-started/importing-pdfs.md

# Importing PDFs

To import a PDF, click on **File -> Import**. A new window will open that allows you to navigate to the folder containing the PDF you want to import.

![](../../../.gitbook/assets/pdf_import_pdf_menu_entry.jpg)

Before iVP actually processes the PDF and imports it, you will be asked to set some parameters that specify how it will be loaded. The options are:

* **Page Size**: Changes the size of the imported PDF, you can either choose from a preset or manually customize the size of the file. Additionally, you can use the dimensions saved in the PDF.

> * **Width Original File**: Displays the original files width, which can be adjusted if the file size isn't taken from it. Also changes the overall width and scale of the PDF when modified.

Reviewer: "Displays the original files width, which can be adjusted if the file size isn't taken from it." This sounds a bit clunky IMO

> * **Width Original File**: Displays the original width of the files, which can be adjusted if the file size isn't derived from it.. Also changes the overall width and scale of the PDF when modified.

Reviewer: ```suggestion
* **Width Original File**: Displays the original width of the file, which can be adjusted if the file size isn't derived from it. Also changes the overall width and scale of the PDF when modified.
```

> * **Scale**: Adjusts the size ratio of the imported file, like scaling it to 50:1, thereby proportionately altering its dimensions.

Reviewer: ```suggestion
* **Scale**: Adjusts the size ratio of the imported file (e.g. 50:1), thereby proportionately altering its dimensions.
```

> In iVP it is possible to import PDFs that will act as an overlay for the [floor plan](../user-interface/the-floor-plan.md). This can be useful for illustration or when it is necessary to transform a 2D (factory) plan into a 3D scene.

Reviewer: This is no longer the case, imported PDFs act as any other object. I think this section can be removed.

## Importing PDFs step by step:

To import a PDF, click on **File -> Import**. A new window will open that allows you to navigate to the folder containing the PDF you want to import.

![](../../../.gitbook/assets/pdf_import_pdf_menu_entry.jpg)

As soon as you have found the file, simply double-click on it or select it and click **Open**.

> ![](../../../.gitbook/assets/pdf_import_pdf_file_dialogue.jpg)

Reviewer: The import works the same for all file types and selecting a file in a file browser probably does not need to be explained. I just explaining where the import button is should be enough.

Before iVP actually processes the PDF and imports it, you will be asked to set some parameters that specify how it will be loaded. The options are:

> * **Page Size**: Changes the size of the imported PDF, you can either choose from a preset or manually customize the size of the file.

Reviewer: You can also use the size that was saved in the Pdf, which is the most important option for our users.

> * **Width Original File**: Displays the width of the original file.

Reviewer: Can also be changed when the page size is not taken from the file. Also explain what this option actually does, when modified.

> * **Scale**: Adjusts the scale of the imported file.

Reviewer: The wording is unclear. We need to make sure we mean scale in the sense of 50:1 (Maßstab)

> * **Resolution**: Adjusts the resolution of the imported file.

Reviewer: Should mention that higher resolution brings sharper lines, but comes with a performance penalty.

> * **Name**: Displays and allows you to change the name of the imported file.

Reviewer: ```suggestion
* **Name**: Modifies the name of the imported object.
```

> * **Add to Library**: Adds and saves the imported item to your [personal library](../user-interface/library-panel.md).

Reviewer: ```suggestion
* **Add to Library**: Saves the imported item to your [personal library](../user-interface/library-panel.md).
```

Author: I wrote "add and save" so it'd be clearer that the imported item can also be used for other projects.

Reviewer: I think just "saved" is enough. The rest is also explained in the linked page.

Author: Okay, will make the change to "saved".
